canbench_results.csv

```diff
@@ -0,0 +1,125 @@
+status,name,instructions,instructions %,heap_increase,heap_increase %,stable_memory_increase,stable_memory_increase %
+,btreemap_contains_key_blob_4_1024,164495877,-0.91%,0,,0,
```

Comment:
do we want to have an empty status?

Reply:
yes, it's fine for now, I don't want to overcomplicate things.
currently it only shows 'new' for added benchmarks. maybe it can also be extended for removed.